resources/js/processes-catalogue/components/ProcessInfo.vue

```diff
@@ -71,6 +113,19 @@ export default {
     this.$root.$on("carouselImageSelected", (pos) => {
       this.firstImage = pos + 1;
     });
+
+    window.ProcessMaker.navbarMobile.display = false;
+  },
+  beforeDestroy() {
+    window.ProcessMaker.navbarMobile.display = true;
```

Comment:
It looks like it can't set this property, in console appears the next message:
vendor.js?id=2f593200eb9b183fc373f1156bd28f7e:119564 [Vue warn]: Error in mounted hook: "TypeError: Cannot set properties of undefined (setting 'display')"

found in

---> <ProcessInfo> at resources/js/processes-catalogue/components/ProcessInfo.vue
       <Process> at resources/js/processes-catalogue/components/Process.vue
         <ProcessesCatalogue> at resources/js/processes-catalogue/components/ProcessesCatalogue.vue
           <Root>